ts/test/scenario.ts

```diff
@@ -346,6 +346,65 @@ test("pauseUntilDhtEqual - Create multiple entries, read the last, 2 conductors"
   await scenario.cleanUp();
 });
 
+test("dhtSync - Fails if some Ops were authored but not yet integrated in a conductor", async () => {
+  const scenario = new Scenario();
+
+  const appBundleSource: AppBundleSource = {
+    type: "path",
+    value: FIXTURE_HAPP_URL.pathname,
+  };
+  const [alice] = await scenario.addPlayersWithApps([
+    { 
+      appBundleSource,
+      options: {
+        networkConfig: {
+          disablePublish: true
+        }
+      },
+    },
+    { appBundleSource },
+  ]);
+
+  // Alice creates 1 entry, but never publishes it because publishing is disabled
+  const actionHash = await alice.cells[0].callZome<EntryHash>({
+    zome_name: TEST_ZOME_NAME,
+    fn_name: "create",
+    payload: "my entry",
+  });
+
+  assert.throws(dhtSync([alice], alice.cells[0].cell_id[0]));
+});
+
+test("dhtSync - Fails if some Ops are not synced among all conductors", async () => {
+  const scenario = new Scenario();
+
+  const appBundleSource: AppBundleSource = {
+    type: "path",
+    value: FIXTURE_HAPP_URL.pathname,
+  };
+  const [alice, bob] = await scenario.addPlayersWithApps([
+    { 
+      appBundleSource,
+      options: {
+        networkConfig: {
+          disablePublish: true,
+          disableGossip: true
```

Comment:
This may not make sense. I could alternatively just have the peers not know about each other by not sharing their peer info or bootstrapping. That would be much simpler than bundling in holochain with `test-utils` enabled.